Makefile

```diff
@@ -49,4 +49,4 @@ shoots/clean:
 	cat % | python src/touch.py staging
 
 %.touched.production: %
-	cat % | python src/touch.py production
+	cat $< | python src/touch.py production
```

Comment:
Couldn't make it work by feeding the list of S3 key directly through the command line so this now takes a file as input like the other commands